sqlglot/dialects/prql.py

```diff
@@ -81,6 +85,36 @@ def _parse_take(self, query: exp.Query) -> t.Optional[exp.Query]:
             num = self._parse_number()  # TODO: TAKE for ranges a..b
             return query.limit(num) if num else None
 
+        def _parse_filter(self, query: exp.Select) -> t.Optional[exp.Query]:
+            if self._match(TokenType.L_PAREN):  # TODO: FILTER ( expr1 || expr2 && ...)
+                filter = self._parse_conjunction()
+                if not self._match(TokenType.R_PAREN):
+                    self.raise_error("Expecting )")
+            else:
+                filter = self._parse_conjunction()
+            return query.where(filter) if filter else None
```

Comment:
Why do we need to separately handle the `(boolean_expression)` case? I think you only need a single `_parse_conjunction` call, the parser will take care of the rest.

So, assuming the above holds, I would just get rid of this whole method and do the following inline:

```python
            "FILTER": lambda self, query: query.where(self._parse_conjunction()),
```